spec/controllers/dashboards_controller_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe DashboardsController do
   describe "#show" do
     it "creates a new SuggestionsJobStatus" do
-      user = FactoryGirl.create(:user)
+      user = create(:user)
       sign_in_as(user)
       p_job_id = double(GetSuggestionsJob, provider_job_id: "67dfd3-5ksTd-d543")
```

Comment:
Maybe use an explanatory variable for this string (and then use it again down on 14)?